java/src/jmri/jmrit/dispatcher/AutoActiveTrain.java

```diff
@@ -2091,7 +2091,9 @@ private void rampToTarget() {
             // target already adjusted.
             log.debug("RampToTarget[{}]current[{}]", getTargetSpeed(), throttle.getSpeedSetting());
             stepQueue = new LinkedList<>();
-            if (throttle.getSpeedSetting() <= getTargetSpeed()) {
+            if (throttle.getSpeedSetting() == getTargetSpeed()) {
```

Comment:
Seeing as 2 floats are being compared, it it worth using a threshold between the 2 values to compare them instead of `==` ?
See https://howtodoinjava.com/java-examples/correctly-compare-float-double/#threshold-based , 
or perhaps use `Float.compare(Float,Float)` https://www.geeksforgeeks.org/float-compare-method-in-java-with-examples/ ?

Reply:
No I dont think so. Any increase or decrease however small must have the limits re-enforced.